util/reggen/validate.py

```diff
@@ -264,21 +263,27 @@ def bitmask(bfield):
     'd': ["int", "integer (binary 0b, octal 0o, decimal, hex 0x)"],
     'x': ["xint", "x for undefined otherwise int"],
     'b': [
-        "bitrange", "bit number as decimal integer, \
-                    or bit-range as decimal integers msb:lsb"
+        "bitrange", "bit number as decimal integer,"
+        "or bit-range as decimal integers msb:lsb"
```

Comment:
This is missing a space after the comma at the end of the 1st line.

Looking down the rest of the file, I think there are the right number of spaces between each pair of strings, but it's a bit hard to read. Maybe it would be better to standardise on either

```python
    "foo bar "
    "baz."
```

or

```python
    "foo bar"
    " baz."
```

throughout? (I prefer the first because otherwise the later lines look oddly indented, but I can see arguments either way).

Reply:
Thanks, will fix.